tools/ci/_util.js

```diff
@@ -0,0 +1,35 @@
+export const ARCHIVE_COUNT = 4;
```

Comment:
Having this const seems brittle, figure out were is it used and can we make it smarter